backend/build.gradle.kts

```diff
@@ -0,0 +1,24 @@
+plugins {
+    java
+    id("org.springframework.boot") version "3.3.9"
+    id("io.spring.dependency-management") version "1.1.4"
+}
+
+group = "org.oaebudt"
+
+java.sourceCompatibility = JavaVersion.VERSION_23
+java.targetCompatibility = JavaVersion.VERSION_23
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation("org.springframework.boot:spring-boot-starter-security")
```

Comment:
importing libraries without specifying the version could lead to problems.
please see my previous comment about gradle version catalog